lib/ansible/modules/system/pamd.py

```diff
@@ -778,7 +778,7 @@ def main():
             backupdest = module.backup_local(fname)
             print("BACKUP DEST", backupdest)
```

Comment:
This isn't part of your changes but if you get time are you able to open a new PR and remove this line. Modules should not be calling `print()` as that can screw up Ansible reading the module output.